ample/util/options_processor.py

```diff
@@ -393,27 +347,52 @@ def process_options(optd):
             optd["side_chain_treatments"] = SIDE_CHAIN_TREATMENTS
     else:
         optd["side_chain_treatments"] = map(str.lower, optd["side_chain_treatments"])
-
     unrecognised_sidechains = set(optd["side_chain_treatments"]) - set(ALLOWED_SIDE_CHAIN_TREATMENTS)
     if unrecognised_sidechains:
         msg = "Unrecognised side_chain_treatments: {0}".format(unrecognised_sidechains)
         logger.critical(msg)
         exit_util.exit_error(msg)
-    #
-    # SCRWL - we always check for SCRWL as if we are processing QUARK models we want to add sidechains to them
-    #
-    #if optd['use_scwrl']:
-    if not optd['scwrl_exe']:
-        optd['scwrl_exe'] = 'Scwrl4' + ample_util.EXE_EXT
-    try:
-        optd['scwrl_exe'] = ample_util.find_exe(optd['scwrl_exe'])
-    except ample_util.FileNotFoundError as e:
-        logger.info("Cannot find Scwrl executable: {0}".format(optd['scwrl_exe']))
-        if optd['use_scwrl']:
-            raise (e)
-    #
+
+    if optd['make_frags']:
+        if optd['use_homs']:
+            logger.info('Making fragments (including homologues)')
+        else:
+            logger.info('Making fragments EXCLUDING HOMOLOGUES')
+    else:
+        logger.info('NOT making Fragments')
+
+    if optd['make_models']:
+        logger.info('\nMaking Rosetta Models')
+    else:
+        logger.info('NOT making Rosetta Models')
+
+
+def process_mr_options(optd):
+    # Molecular Replacement Options
+    if optd['molrep_only']:
+        optd['phaser_only'] = False
+    if optd['molrep_only']:
+        optd['mrbump_programs'] = ['molrep']
+    elif optd['phaser_only']:
+        optd['mrbump_programs'] = ['phaser']
+    else:
+        optd['mrbump_programs'] = ['molrep', 'phaser']
+    if optd['phaser_rms'] != 'auto':
+        try:
+            phaser_rms = float(optd['phaser_rms'])
+            optd['phaser_rms'] = phaser_rms
```

Comment:
Use again the try - except - else statement structure. Assignment is not really part of the try - except clause